### DISCUSSION TITLE
Is there a way to prevent Renovate from recommending non semver-compatible dependencies?

### DISCUSSION THREAD

**Discussion by danielbankhead** [renovatebot/renovate#24679] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

If I'm on Node 14, I don't want renovate to recommend Node 16+ dependencies until I've migrated to Node 16. Today, we constantly ignore breaking change PRs or accidentally merge them causing unexpected issues for us and our customers.

### What you tried so far.

I've looked at the docs, github issues, and discussions, couldn't find anything related to this (or, I didn't see/use the proper terms).

**Comment by rarkins** [comment]
Your description doesn't match the title at all. Why do you think packages supporting Node 16 aren't semver compatible?

**Reply by rarkins** [reply]
If I were to ignore the title and answer the description then it sounds like you're after the option constraintsFiltering. That one surprisingly hard to get right though as there isn't a single convention for how libraries should specify support (e.g. do you skip the odd node versions, etc)
